src/locales/locale-en.json

```diff
@@ -154,6 +155,11 @@
     "USERPROFILE": "Visit your profile",
     "CHANGE_GROUP": "Change group"
   },
+  "SIDENAV": {
+    "INFO_SUB": "How to build your foodsaving group",
+    "BLOG_SUB": "All News around karrot.world",
```

Comment:
should be "news" not "News"?

Reply:
Ahaha that's what I was thinking! But in the older version from you it was written like that so I simply kept it that way...^^